src/Compilers/CSharp/Portable/Symbols/Source/ThisParameterSymbol.cs

```diff
@@ -204,5 +204,7 @@ static bool hasUnscopedRefAttribute(MethodSymbol? containingMethod)
                 }
             }
         }
+
+        internal sealed override bool UseUpdatedEscapeRules => _containingMethod?.UseUpdatedEscapeRules ?? false;
```

Comment:
Why would we not check `_containingType` when `_containingMethod` is null?

Reply:
Good catch, thanks.